aspnetcore/fundamentals/host/generic-host.md

```diff
@@ -223,9 +223,491 @@ Host.CreateDefaultBuilder(args)
 
 If the timeout period expires before all of the hosted services stop, any remaining active services are stopped when the app shuts down. The services stop even if they haven't finished processing. If services require additional time to stop, increase the timeout.
 
-**Key**: shutdownTimeoutSeconds  
-**Type**: *int*  
-**Default**: 5 seconds
+**Key**: `shutdownTimeoutSeconds`  
+**Type**: `int`  
+**Default**: 5 seconds  
+**Environment variable**: `<PREFIX_>SHUTDOWNTIMEOUTSECONDS`
+
+To set this value, use the environment variable or configure `HostOptions`. The following example sets the timeout to 20 seconds:
+
+[!code-csharp[](generic-host/samples-snapshot/3.x/Program.cs?name=snippet_HostOptions)]
+
+### Disable app configuration reload on change
+
+By default, an app configuration file such as *appsettings.json* is reloaded when the file changes. To disable this reload behavior in ASP.NET Core 5.0 Preview 3 or later, set the `hostBuilder:reloadConfigOnChange` key to `false`.
```

Comment:
```suggestion
By [default](xref:fundamentals/configuration/index#default), *appsettings.json* and *appsettings.*`Environment`*.json* are reloaded when the file changes. To disable this reload behavior in ASP.NET Core 5.0 Preview 3 or later, set the `hostBuilder:reloadConfigOnChange` key to `false`.
```

Reply:
See https://review.docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-3.1&branch=pr-en-us-17152#default-configuration which should merge soon.